flink-java/src/main/java/org/apache/flink/api/java/typeutils/PojoTypeInfo.java

```diff
@@ -310,6 +310,9 @@ public int getFieldIndex(String fieldName) {
 
 	@Override
 	public TypeSerializer<T> createSerializer(ExecutionConfig config) {
+		if(config.isForceKryoEnabled()) {
+			return new GenericTypeInfo<T>(this.typeClass).createSerializer(config);
```

Comment:
I thought it makes sense to not have the switch, since the flag is called
"forceKryo"...